src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -90,7 +90,7 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                 </ngx-datatable-column>
                 <ngx-datatable-column name="Description" prop="description">
                     <ng-template ngx-datatable-cell-template let-row="row">
-                        <textarea class="form-control table-editable-field__input me-2" [id]="row.id + '-description'" [(ngModel)]="row.description"> </textarea>
+                        <textarea class="form-control table-editable-field__input me-2" [id]="row.id + '-description'" [(ngModel)]="row.description" name="description"> </textarea>
```

Comment:
One question: Why is this change needed? :thinking: 

Edit: Seems like thats the bug fix right?

Reply:
Yes, that was needed in order to save the description correctly.